### DISCUSSION TITLE
translate to spanish

### DISCUSSION THREAD

**Discussion by julbroa** [te-online/timemanager#199] [General]
¿how can i to help to translate to spanish?

**Comment by te-online** [comment]
Hey! Thanks for offering 😊 

Right now it's quite a manual process. I recommend you to fork this repository to your own Github account. Then you can update files. You need to [copy one of the more exhaustive translation files in this folder](https://github.com/te-online/timemanager/tree/main/l10n) from another language, e.g. `fr.js` and `fr.json` or `de.js` and `de.json`. There always needs to be a `.js` and `.json` version that have the same translations, because one is for the frontend and one is for the backend.

When you're done on your forked repository, you can create a Pull Request (PR) to this repository, so that I can merge the changes you made on your end.

Hope it makes sense, let me know if you have questions 😊
